src/Rector/Return_/CommandConstantReturnCodeRector.php

```diff
@@ -82,19 +82,31 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if (! $classReflection->hasMethod('execute')) {
+        if (! $this->nodeNameResolver->isName($node, 'execute')) {
             return null;
         }
 
-        if (! $node->expr instanceof LNumber) {
-            return null;
-        }
-
-        $node->expr = $this->convertNumberToConstant($node->expr);
+        $this->findReturnStatement($node->stmts);
```

Comment:
The `Return_` to be found must be scoped, not under Closure or inner function, so you need to use : 

```
$this->betterNodeFinder->findInstancesOfInFunctionLikeScoped($node, [Return_::class]);
```

Reply:
Will have a look again to see if i can fix it with `findInstancesOfInFunctionLikeScoped`

Reply:
Only `replace_with_constant_nested_function.php.inc` doesn't work as the `expr` on line 95 is a `MethodCall` to `executeCommandStuff`

Reply:
@samsonasik maybe you can give a hint?

Reply:
@johanadivare that's because you already have this skip:

```php
        if (! $this->nodeNameResolver->isName($node, 'execute')) {
            return null;
        }
```

that make it skipped, as the method you want to change the return value is different method.

You may need separate Rector rule for that, that will be easier to verify the method is called in `Class_`->getMethod('execute')`

Reply:
Okay fair point, i will remove the part checking for nested functions